src/Parser/Cif.php

```diff
@@ -363,7 +363,9 @@ private static function parseXmlNode(\Gettext\Translations $translations, $filen
                 static::readXmlNodeAttribute($translations, $filenameRel, $node, 'custom-label', 'PageTypeComposerFormLayoutSetControlCustomLabel');
                 break;
             default:
-                throw new \Exception('Unknown tag name '.$path.' in '.$filenameRel."\n\nNode:\n".$node->ownerDocument->saveXML($node));
+                if (strpos($filenameRel, 'packages/') !== false) {
```

Comment:
`!== false` means "it contains `packages/`". I think it would be better to have `!== 0`, ie "it does not start with `packages/`"